src/app/index.ts

```diff
@@ -0,0 +1,1003 @@
+
+import { get } from 'https';
+import Products from './product';
+
+let products: any = [];
+let loadingQuantity = 4;
+let catalogLength: any = null;
+let filteredProducts: any[] = [];
+const catalogList = document.querySelector(".catalog-list");
+const catalogListPopup = document.querySelector(".catalog-popup");
+const catalogLoadMore = document.querySelector(".catalog__more");
+
```

Comment:
it has made sense to think about providing mechanisms for splitting JavaScript programs up into separate modules that can be imported when needed. There are a lot of benefits to using modules in favor of a sprawling, interdependent codebase